src/Symfony/Bundle/FrameworkBundle/Test/WebTestAssertionsTrait.php

```diff
@@ -30,12 +30,12 @@ trait WebTestAssertionsTrait
 {
     public static function assertResponseIsSuccessful(string $message = ''): void
     {
-        self::assertThat(static::getResponse(), new ResponseConstraint\ResponseIsSuccessful(), $message);
+        self::assertThat(self::getResponse(), new ResponseConstraint\ResponseIsSuccessful(), $message);
```

Comment:
Traits should be self contained, we shouldn't provide these as extensibility points.